src/Xamarin.Android.NUnitLite/Xamarin.Android.NUnitLite.csproj

```diff
@@ -360,4 +97,16 @@
       <Private>False</Private>
     </ProjectReference>
   </ItemGroup>
+  <Target Name="ILRepacker"
```

Comment:
This should have `Inputs` and `Outputs` so that it doesn't needlessly re-run on every Build when nothing has changed.

Reply:
This target should also be moved into a new `Xamarin.Android.NUnitLite.targets` file, because Visual Studio for Mac has a habit of reformatting `.csproj` files whenever you add/remove/alter the `.sln`, which means all the wonderful whitespaces and readability here will be "helpfully" removed.

VSMac *doesn't* process `.targets` files in this fashion, which is why we (usually) place our custom build logic into `.targets` files which are `<Import/>`ed from the `.csproj`.

Reply:
I can't put an `Inputs` and `Outputs` because for some reasons, `Xamarin.Android.NUnitLite.dll` gets overwritten in the `AfterBuild` target, but the `Inputs` doesn't detect it. This ends up messing with `Xamarin.Android.NUnitLite.dll` on a rebuild, leading to it not containing `nunitlite.dll`. The only solution I found was to run it every time, so we make sure it gets merged properly.

The chain of even is as following:

1. Build with `make all`
   - `CoreCompile` compiles `Xamarin.Android.NUnitLite.dll` because `$(IntermediateOutputPath)/Xamarin.Android.NUnitLite.dll` doesn't exist; it also references `nunitlite.dll` to make sure it compiles successfully.
   - `CopyToOutputBlahBlah` copies `$(IntermediateOutputPath)/Xamarin.Android.NUnitLite.dll` to `$(OutputPath)/`
   - `ILRepacker` merges `nunitlite.dll` into `$(OutputPath)/Xamarin.Android.NUnitLite.dll`.
1. Rebuild with `make all`
   - `CoreCompile` does nothing because `$(IntermediateOutputPath)/Xamarin.Android.NUnitLite.dll` Inputs and Outputs hasn't changed
   - `CopyToOutputBlahBlah` copies `$(IntermediateOutputPath)/Xamarin.Android.NUnitLite.dll` to `$(OutputPath)/`
   - `ILRepacker` with `Inputs=$(OutputPath)/Xamarin.Android.NUnitLite.dll` *doesn't* detect the _new_ `$(OutputPath)/Xamarin.Android.NUnitLite.dll` and skips the target. <-- breakage! `$(OutputPath)/Xamarin.Android.NUnitLite.dll` hasn't merge `nunitlite.dll`.

On closer inspection, *after* a rebuild, `$(OutputPath)/Xamarin.Android.NUnitLite.dll` still has the same modification time as *before* the merge, so my guess is `CopyToOutputBlahBlah` copies the timestamp at the same time as the content from `$(IntermediateOutputPath)/Xamarin.Android.NUnitLite.dll`.

I will move it to `Xamarin.Android.NUnitLite.targets`.